src/coreclr/runtime.proj

```diff
@@ -58,8 +58,8 @@
       <_CoreClrBuildArg Condition="'$(ClrCrossComponentsSubset)' == 'true'" Include="-component crosscomponents" />
     </ItemGroup>
 
-    <ItemGroup Condition="'$(TargetsAppleMobile)' == 'true'">
-      <_CoreClrBuildArg Include="-cmakeargs -DCMAKE_ICU_DIR=&quot;$(_IcuDir)&quot;" />
+    <ItemGroup Condition="'$(TargetsAppleMobile)' == 'true' or '$(TargetArchitecture)' == 'wasm'">
+      <_CoreClrBuildArg Include="-cmakeargs &quot;-DCMAKE_ICU_DIR=$(_IcuDir)&quot;" />
```

Comment:
Is the quoting change intentional?

Reply:
Yes, it was.  I had problems with the value getting dropped, but wasn't sure why.  With the quoting as it was I get

```
Calling "E:\GitHub\runtimelab\eng\native\gen-buildsys.cmd" "E:\GitHub\runtimelab\src\coreclr" "E:\GitHub\runtimelab\artifacts\obj\coreclr\browser.wasm.Debug" vs2022 wasm browser "-DRUNTIME_FLAVOR=CoreCLR" "-DCMAKE_BUILD_TYPE=Debug" "-DCLR_CMAKE_KEEP_NATIVE_SYMBOLS=1" "-DCLR_CM
  AKE_TARGET_ARCH=wasm" "-DCLR_CMAKE_TARGET_OS=emscripten" "-DCLI_CMAKE_FALLBACK_OS=browser" "-DCLR_CMAKE_PGO_INSTRUMENT=0" "-DCLR_CMAKE_OPTDATA_PATH=" "-DCLR_CMAKE_PGO_OPTIMIZE=0" -DCMAKE_ICU_DIR
```
You can see `-DCMAKE_ICU_DIR` had no value, I wasn't sure if it was the backslash  in `c:\` or something else.

Reply:
And ass the other `-D` arguments were quoted to include the `-D...` I did the same.